test/compilable/test13582b.d

```diff
@@ -0,0 +1,14 @@
+// REQUIRED_ARGS: -de
```

Comment:
Output check ?

Reply:
For what? There's not supposed to be any output, and if there's any it'd be an error and fail that way.

Reply:
Yeah I somehow didn't register they were in compilable, my bad.